airflow-core/src/airflow/api_fastapi/core_api/datamodels/ui/common.py

```diff
@@ -79,6 +79,9 @@ class GridRunsResponse(BaseModel):
     run_after: datetime
     state: DagRunState | None
     run_type: DagRunType
+    dag_version_number: int | None = None
```

Comment:
I don't think you need to pass this down, because the grid is calling `get_dag_details` and the latest `dag_version` is already there in the payload.

So you might get it in the front-end from there. same for the bundle name and bundle version.